docker-storage-setup.sh

```diff
@@ -823,6 +830,13 @@ setup_storage() {
   fi
 }
 
+reset_storage() {
+    if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
+	reset_lvm_thin_pool
+    fi
+    rm -f $DOCKER_STORAGE
```

Comment:
This is one of those things where I feel the push from some groups to have an empty `/etc` by default sounds good in theory, but in practice would be painful.  There's a massive amount of precedence for having stubbed out configuration files in `/etc`.  It greatly aids "discoverability".

I think deleting it now is not such a good idea.  But I'm not strictly opposed to it either.